vite.config.js

```diff
@@ -62,6 +62,7 @@ export default defineConfig({
         skipWaiting: true, // Force the waiting service worker to become the active service worker
         clientsClaim: true, // Take control of uncontrolled clients as soon as the service worker becomes active
         maximumFileSizeToCacheInBytes: 6000000, // 6MB
+        navigateFallbackDenylist: [/^\/eapi\/.+/], // Prevent caching of external API routes
```

Comment:
am not sure why we need this ? for `eapi`

Reply:
Because the service worker serves requests from cache and we end up being served the 404 error page for API routes.
The only way to have the actual API response when directly accessing an API endpoint is by doing a hard refresh in the browser. It's the same for `/api`endpoint by the way (you can try the `/api/v1/resource` endpoint in your browser) but since `/eapi` is supposed to be an external API I think it shouldn't be cached.

I was playing with the `/eapi` route but I changed my strategy so I won't be using it in the end, if you don't want this change it's fine for me.